frontend/lint/tests/util.rs

```diff
@@ -0,0 +1,39 @@
+use std::process::Command;
+
+use regex::Regex;
+
+pub struct Test {
+    pub stderr: &'static str,
+    pub manifest_path: &'static str,
+}
+
+/// We need to install the driver before we can run the circus command.
+/// "cargo install --path cli/driver"
+pub fn install_driver() {
+    let mut cmd = Command::new("cargo");
+    cmd.current_dir("../../");
+
+    let status = cmd
+        .args(&["install", "--path", "cli/driver"])
+        .output()
+        .unwrap()
+        .status;
+    assert!(status.success());
+}
```

Comment:
I think we should not install a binary while running test, it's a side effect I would not expect when running tests locally

Reply:
I agree, but this required rewriting the binaries. `cargo-circus` currently calls the driver binary, which requires that it's installed. This is generally bad, also for local development because you end up getting an old installed version of part of the repo when using `cargo run`.
If you want to fix this before getting this in, we can do that. But otherwise I'd prefer getting this merged and then getting it fixed in a follow up.

Reply:
Yes, so I fixed it in 42c2744c8fb806ebc10baaeeb2dc8d75db5947c1 by adding an environment variable the circus driver is sensible to, so that we can inject the absolute path to the local cargo binary

If that change is fine to you, feel free to cherry-pick it in this branch :) (I can also do it if you want)